app/ceom/maps/models.py

```diff
@@ -1,6 +1,7 @@
 from django.db import models
 from ceom.photos.models import Category
 from django.contrib.auth.models import User
+from django.contrib.gis.db import models
```

Comment:
You're importing two different things on lines 1 and 4 and naming them both "models"
I think in this case you can delete the import on line 1 since its being overwritten by line 4

Try deleting line 1 and see if everything still works